### ISSUE TITLE
windows 10 에서 CP949 mode 이상 동작

### ISSUE THREAD

**Issue by Joungkyun** [teamnop/HPuTTY#16]
windows 10에서 unicode 모드를 해제하고 (CP949 모드로 전환), 문자셋을 ko_KR.eucKR 로 설정을 하더라도, CP949 문자가 깨져서 나오는 문제가 있음.

![capture1](https://cloud.githubusercontent.com/assets/7565963/25348474/5c1001ae-2959-11e7-90b5-3790a0db2d90.jpg)
![capture2](https://cloud.githubusercontent.com/assets/7565963/25348476/5feb42ac-2959-11e7-9932-dea7b0028a06.jpg)
![capture3](https://cloud.githubusercontent.com/assets/7565963/25348478/61fab1b8-2959-11e7-9bdb-ff4ec819bc66.jpg)
![capture4](https://cloud.githubusercontent.com/assets/7565963/25348482/63e624da-2959-11e7-85f0-ea951b8ce59e.jpg)

**Comment by noricube** [comment]
UTF-8로만 써서 몰랐던 문제네요. 한번 확인해보겠습니다.

**Comment by Joungkyun** [comment]
음. 무언가 경우의 수가 있는 것 같군요. 정상적으로 동작하는 machine도 있고 안되는 machine도 있네요.
안되는 machine에서는 CP949 대신 "Use font encoding"을 선택하니까 되기는 합니다.

**Comment by noricube** [comment]
제가 사용하는 machine에서도 재현이 되긴하네요. Use font encoding을 사용하면 된다고 하시는것 보니 뭔가 설정 입력의 문제인것 같기도 하고요.
테스트 해보면서 알게 된 것인데 저는 IME에도 문제가 있어 보이네요.

참고해서 한번 보도록 하겠습니다.

**Comment by Joungkyun** [comment]
네 입력기 문제도 분명히 있습니다. 재현이 되는 컴에서는 unicode mode가 아닐 경우 한글 입력도 되지 않거든요.

**Comment by noricube** [comment]
몇가지 테스트 해보니 CP949로 바꾼 설정이 제대로 적용되지 않는 문제인것 같습니다.
확인하고 수정하면 이 issue에 추가로 댓글 달겠습니다.

이 문제와 별도로 IME 문제는 좀 더 찾아봐야 될 것 같습니다.

**Comment by Joungkyun** [comment]
흠 일단 문제는 폰트 설정에 있는 것 같습니다. 폰트 설정에서 스크립트가 "한글"로 설정이 되어 있지 않을 경우에 한글 입출력이 되지 않더군요.
![font](https://cloud.githubusercontent.com/assets/7565963/26029392/b52f8f10-386e-11e7-8483-12fade745665.JPG)

iputty 에서는 https://github.com/Joungkyun/iputty/commit/34517c5f483aa5e909202003e2eb08830e6000f1 로 처리를 했습니다.

**Comment by noricube** [comment]
음 그렇네요. 확인해보니 제가 기본 폰트로 지정해둔 Consolas는 HANGUEL_CHARSET만 지원해서 그런것 같네요.
실제로 수정하신것으로 테스트 해보니 잘 되는것 확인했습니다.
